components/common/CustomInput/CustomInput.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+
+import { useRef } from "react";
+import useInputValidation from "hooks/useInputValidation";
+import InputBox from "./InputBox";
+import UserInput from "./UserInput";
+
+import styles from "./CustomInput.module.scss";
+
+type ValidationType = {
+  email: string,
+  password: string,
+  checkingPassword?: string
+};
```

Comment:
```suggestion
interface IValidation {
  email: string,
  password: string,
  checkingPassword?: string
};
```

컨벤션 정할 때 객체타입은 interface로 선언하기로 했습니다.